ProffieOS.ino

```diff
@@ -16,10 +16,16 @@
 
  You should have received a copy of the GNU General Public License
  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+
+-------------------------------------------------------------------
+You can have multiple configuration files, and specify which one
```

Comment:
I think I like the text update, but I think the copyright blurb should be in a separate comment.

Reply:
ok how about now?